base/uk.ac.stfc.isis.ibex.e4.ui.perspectiveswitcher/src/uk/ac/stfc/isis/ibex/e4/ui/perspectiveswitcher/views/PerspectiveSwitcherView.java

```diff
@@ -1,7 +1,6 @@
 package uk.ac.stfc.isis.ibex.e4.ui.perspectiveswitcher.views;
 
 import java.util.Collections;
-import java.util.Comparator;
 import java.util.List;
```

Comment:
This import wasn't used and was causing a checkstyle error.

Reply:
👍